### PR TITLE
Remove the option to set height

### LINKED ISSUE #6541: Resized images cause distortion
#### Description
.bbc.resized on phone sized devices seem to cause a distortion and look bad

See this sample image
Screen Shot 2021-02-26 at 10 16 44 AM

The fix I have found that works is to instead of setting max height/width.  I set the height/width to auto.  This seems to scale the image properly so it doesn't look distorted.

Any unforeseen issues with this fix?

### PR DESCRIPTION
**Pull request by BrickOzp**
Remove the option to set height for images and embedded
attachments. Having a set height when the width is
responsive, causes aspect ratio to be wrong in some cituations.

Fixes #6541

Signed-off-by: Oscar Rydhé <oscar.rydhe@gmail.com>

### CODE REVIEW

**Comment by BrickOzp** [comment]
This is a radical idea.
See investigation in https://github.com/SimpleMachines/SMF2.1/issues/6541

File: Sources/Subs.php [hunk 1]
```diff
@@ -1750,9 +1747,9 @@ function parse_bbc($message, $smileys = true, $cache_id = '', $parse_tags = arra
 					'alt' => array('optional' => true),
 					'title' => array('optional' => true),
 					'width' => array('optional' => true, 'value' => ' width="$1"', 'match' => '(\d+)'),
-					'height' => array('optional' => true, 'value' => ' height="$1"', 'match' => '(\d+)'),
+					'height' => array('optional' => true, 'value' => '', 'match' => '(\d+)'),
 				),
-				'content' => '<img src="$1" alt="{alt}" title="{title}"{width}{height} class="bbc_img resized" loading="lazy">',
+				'content' => '<img src="$1" alt="{alt}" title="{title}"{width} class="bbc_img resized" loading="lazy">',
```
Comment (albertlast) [inline Sources/Subs.php]: google and user didn't like lazy load without height.

Comment (BrickOzp) [reply Sources/Subs.php]: Fixed

**Comment by DiegoAndresCortes** [comment]
It's alright imo.
I provided my thoughts on the other pr, so whatever is decided it's fine by me, we just need to be aware that it needs feedback from the average users to understand what behavior they expect.

**Pull request closed without merging**